src/components/organisms/ProfileInfo/index.jsx

```diff
@@ -0,0 +1,324 @@
+//#region > Imports
+//> React
+// Contains all the functionality necessary to define React components
+import React from "react";
+//> MDB
+// "Material Design for Bootstrap" is a great UI design framework
+import {
+  MDBBtn,
+  MDBBadge,
+  MDBView,
+  MDBMask,
+  MDBPopover,
+  MDBPopoverBody,
+  MDBPopoverHeader,
+  MDBIcon,
+  MDBTooltip,
+} from "mdbreact";
+//#endregion
+
+//#region > Components
+class ProfileInfo extends React.Component {
```

Comment:
Add description: `This component displays personal information and status of a user.`